client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -36,23 +36,23 @@
 @Slf4j
 public class FlowHandler extends ChannelInboundHandlerAdapter implements AutoCloseable {
 
-    private static final int FLOW_DISABLED = -1;
+    private static final int FLOW_DISABLED = 0;
```

Comment:
What was wrong with -1?

Reply:
if the flow id is disabled ; the request id should start from the value :  (-1<<32) (-1, 0). This could be confusing.. so, if the flow id is disabled , the user of flow handler need not be aware that initial value should be (-1<<32) rather it can be just 0.
it was no wrong with having -1 , 0 is better  to understand avoids bugs in case flow id is disabled.